packages/rocketchat-api/server/routes.coffee

```diff
@@ -203,37 +255,42 @@ RocketChat.API.v1.addRoute 'user.update', authRequired: true,
 		catch e
 			return RocketChat.API.v1.failure e.name + ': ' + e.message
 
+
 # Get User Information
 RocketChat.API.v1.addRoute 'user.info', authRequired: true,
 	post: ->
 		if RocketChat.authz.hasRole(@userId, 'admin') is false
 			return RocketChat.API.v1.unauthorized()
-
 		return { user: RocketChat.models.Users.findOneByUsername @bodyParams.name }
 
+
 # Get User Presence
 RocketChat.API.v1.addRoute 'user.getpresence', authRequired: true,
 	post: ->
 		return { user: RocketChat.models.Users.findOne( { username: @bodyParams.name} , {fields: {status: 1}} ) }
 
-# Delete User
-RocketChat.API.v1.addRoute 'users.delete', authRequired: true,
```

Comment:
This could be a breaking change.  @rocketchat/core do we want to keep this method naming? or at least keep it with some feedback as to why its not working instead of it just disappearing?